contracts/strategies/stkCvxFXS/stkCvxFxsVault.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "../../GenericVault.sol";
+
+struct EarnedData {
+    address token;
+    uint256 amount;
+}
+
+interface stkCvxFxsStrategy {
+    function harvest(address _caller, uint256 _minAmountOut)
+        external
+        returns (uint256 harvested);
+}
+
+interface ICvxFxsStaking {
+    function claimableRewards(address _account)
+        external
+        view
+        returns (EarnedData[] memory userRewards);
+}
+
+contract stkCvxFxsVault is GenericUnionVault {
+    bool public isHarvestPermissioned = false;
+    mapping(address => bool) public authorizedHarvesters;
+
+    constructor(address _token) GenericUnionVault(_token) {}
```

Comment:
The underlying is already hardcoded in the strategy contract (see below) so the same should be done here, otherwise it wouldn't function.

Hardcoded examples
- https://github.com/convex-community/union_contracts/pull/20/files#diff-0ec1aaaf52525276ff969e8a3a62feb42665645eaecf44d2bab151939adebf88R15
- https://github.com/convex-community/union_contracts/pull/20/files#diff-0ec1aaaf52525276ff969e8a3a62feb42665645eaecf44d2bab151939adebf88R56